MahakPal/Day4/01.c

```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
+
+
+int factorial(int n){
+
+if (n == 0){
+   return 1;
+   } else {
+return n * (n-1);
+   }
+}
+int main()
+{
+    int number;
+    printf("Enter any number\n");
+    scanf("%d", &number);
+
+    printf("%d", number, factorial);
```

Comment:
You have not called the function properly and also if user input negative number how will you handle it.